code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -640,7 +721,13 @@ var/global/list/robot_modules = list(
 					LANGUAGE_SIGN	 = 0
 					)
 	sprites = list(
-					"Dread" = "securityrobot",

```

Comment:
This was oddly used in syndicate sprites despite being also used in security sprites. I removed it to replace it with actual antag sprites.